v1/pkg/client/api/openapi.yaml

```diff
@@ -5118,7 +5131,12 @@ components:
           description: Indicates whether or not the fabric dedicated for private networks
             supports VNI ID.
           type: boolean
+        ControllerID:
+          description: Unique ID for the controller that manages the data center.
+          format: uuid
+          type: string
       required:
+      - ControllerID
```

Comment:
What if we are early in Day -1 where we have a Hoster (Tenant) and Pod (Site) in the DB but has not gone to factory yet, so we have no controller attached?

Reply:
It will return empty string, we started to use required after the "omitempty" discussion with GUI team.